### PR TITLE
Start move of rhino3dm, .compute, .inside samples

### PR DESCRIPTION
**Pull request by fraguada**
As discussed previously, we will move samples from rhino3dm, rhino.inside, and rhino.compute projects here.

This is a first pass, including rhino3dm and rhino.compute samples, but includes the directory structure for the rest of the work. 

Samples have been renamed according to the standards in this repository. @dalefugier Please review to ensure consistency with the rest of the samples and comment on anything you see out of place.

### CODE REVIEW

File: rhino.compute/js/SampleBrepBoolean/index.html [hunk 1]
```diff
@@ -0,0 +1,559 @@
...
+    init();
+    animate();
+
+    /**
+     * Set up empty GL canvas with a gradient background. The geomtry will
+     * be loaded into the scene through a separate routine.
+     */
+    function init() {
+        RhinoCompute.authToken = RhinoCompute.getAuthToken();
+        RhinoCompute.url = 'https://compute.rhino3d.com/';
```
Comment (pearswj) [inline rhino.compute/js/SampleBrepBoolean/index.html]: This should be updated to point to localhost (perhaps the compute.rhino3d library should default to localhost itself...). Also, could we standardise the way that these samples are structured to make it easier to see the differences between them? I know I'm picking on a particularly old sample here that contains some helper methods that are no longer necessary...

File: rhino.compute/py/SampleTkinter/makemesh.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    top.destroy()
+lbl = Tkinter.Label(top, text="Rhino Accounts auth token")
+lbl.grid(row=0, column=0)
+auth = Tkinter.Entry(top)
+auth.grid(row=0, column=1)
+btn = Tkinter.Button(top, text='Ok', command=onClickOk)
+btn.grid(row=1, column=1)
+top.mainloop()
+
+compute_rhino3d.Util.authToken = auth_token
```
Comment (pearswj) [inline rhino.compute/py/SampleTkinter/makemesh.py]: Another instance where the auth token and url need to be updated.

**Review by pearswj — COMMENTED** [review]
Thanks for getting this rolling!

This might seem like splitting hairs, but Compute gets called many things. Do we use "rhino.compute" anywhere else? What about simply naming the directory "compute" to match the naming of the section on devdocs (https://developer.rhino3d.com/guides/compute/)?

I also left a couple of line comments about updating the samples to use localhost and not use auth tokens. That might be premature and could be fixed later!

File: rhino.compute/js/SampleBrepBoolean/index.html (hunk 1 above)
Comment (fraguada) [reply rhino.compute/js/SampleBrepBoolean/index.html]: I did change a few to use localhost as I was moving them, though I plan to do a second pass to review the code in all of these. I agree that some of these samples need to be updated. 

I think the compute client libraries should default to localhost.

Comment (pearswj) [reply rhino.compute/js/SampleBrepBoolean/index.html]: Sounds good!

File: rhino.compute/py/SampleTkinter/makemesh.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
+from rhino3dm import *
+import tkinter as Tkinter
+import time
+import compute_rhino3d.Mesh
+import compute_rhino3d.Util
+
+auth_token = ""
+top = Tkinter.Tk()
+def onClickOk():
```
Comment (fraguada) [inline rhino.compute/py/SampleTkinter/makemesh.py]: Can this just be deleted?

Comment (pearswj) [reply rhino.compute/py/SampleTkinter/makemesh.py]: Yep! Everything down to and including `compute_rhino3d.Util.authToken = auth_token` can be chucked.

**Comment by fraguada** [comment]
I've moved the Rhino.Inside dotnet and Javascript sample. I will hold off adding other Rhino.Inside samples in this PR as they should all be tested and the ones remaining will take a bit longer for me to go through. We should also update the Rhino Inside dotnet samples to net48 and the updated Resolver. Next PR

**Review by pearswj — APPROVED** [review]
Looks great to me!

**Pull request merged**